tests/Processor.IntegrationTests/Consumers/ResourceEventsConsumerTests.cs

```diff
@@ -66,7 +72,10 @@ public async Task WhenDecisionNotificationSent_ThenDecisionNotificationIsSentToA
 
         await SendMessage(
             MRN,
-            JsonSerializer.Serialize(resourceEvent),
+            JsonSerializer.Serialize(
+                resourceEvent,
+                new JsonSerializerOptions() { PropertyNamingPolicy = new UpperCaseNamingPolicy() }
```

Comment:
Even thought we will be undoing this change, we might as well address the sonar warning and create a single static `JsonSerializerOptions` instance